pkgs/tools/admin/pebble/default.nix

```diff
@@ -1,26 +1,27 @@
-{ buildGoPackage
+{ buildGoModule
 , fetchFromGitHub
 , lib
 }:
 
 let
-  version = "v2.2.2";
+  version = "v2.3.0";
   pname = "pebble";
-in buildGoPackage {
+in buildGoModule {
   inherit pname version;
-  goPackagePath = "github.com/letsencrypt/${pname}";
 
   src = fetchFromGitHub {
     owner = "letsencrypt";
     repo = pname;
     rev = version;
-    sha256 = "10g6ivdxxp3632wk0gvmp75v9x668kchhmlczbsq8qnsc8sb8pwf";
+    sha256 = "1piwzzfqsdx6s2niczzp4mf4r3qn9nfdgpn7882g52cmmm0vzks2";
   };
 
+  modSha256 = "0n5s5y2gnz4lrla7a52lgmnzkqda1dblhnmv069zcvdfwmf79ac9";
```

Comment:
Why change to `buildGoModule` if `buildGoPackage` with the vendored dependencies still work?   Also see https://github.com/NixOS/nixpkgs/issues/84826

Reply:
I'm not too opinionated about it; but other people seem to be very against introducing any new `buildGoModule` things into `nixpkgs`

Reply:
I tend to agree with the linked issue and dislike fixed-output derivations, was just following the manual's recommendation for best practices. Happy to change this back to `buildGoPackage`.

Reply:
I remember why now: `buildGoPackage` doesn't support the official Go module dependency infrastructure at all, which seems unfortunate, although not a blocker in this case. What confuses me is that there was no vendored `deps.nix` even before 4e408d9d11d71fd8e0a15121bda7d58f1bdcc507...

Reply:
Now uses `buildGoPackage` + vgo2nix.

@GrahamcOfBorg test acme acme-dns

Reply:
I think it's because they have everything in a `vendor` folder

Reply:
Oh, huh... Go is so weird. Dose this mean that the `deps.nix` is incorrect?

Reply:
Because they could be afraid a dependency out of their control disappear from GitHub;)
So, in this case, I think `goBuildPackage` without any `deps.nix` file should do the job.